go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/urfave/cli
+
+go 1.12
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	gopkg.in/urfave/cli.v1 v1.20.0
```

Comment:
This is back

Reply:
I think that is there due to the package being referenced in the flag type in the `altsrc` package. We would need to find a way to eventually merge it into the main package and get rid of this too in the process.